app/src/main/res/values/faqs.xml

```diff
@@ -9,13 +9,13 @@
+  <string name="faq_answer_6"><![CDATA[ <span> If the Exploration Player is not loading </span> <br/><br/> <span>Check to see if the app is up to date: </span><ul><li dir="rtl"> Go to the Play Store and make sure the app is updated to its latest version </li> </ul> <span>Check your internet connection: </span><ul><li dir="rtl"> If your internet connection is slow, try re-connecting to your Wi-Fi network or connecting to a different network. </li> </ul>  <span>Ask the Administrator to check their device and internet connection: </span><ul><li dir="rtl"> Get the Administrator to troubleshoot using the steps above </li> </ul>  <span>Let us know if you still have issues with loading: </span><ul><li dir="rtl"> Report a problem by contacting us at admin@oppia.org. </li> </ul>]]></string>
+  <string name="faq_answer_7"><![CDATA[ <span> If your audio is not playing </span> <br/><br/> <span>Check to see if the app is up to date: </span><ul><li dir="rtl"> Go to the Play Store and make sure the app is updated to its latest version </li> </ul>  <span>Check your internet connection: </span><ul><li dir="rtl"> If your internet connection is slow, try re-connecting to your Wi-Fi network or connecting to a different network. Slow internet may cause the audio to load irregularly, making it difficult to play. </li> </ul>  <span>Ask the Administrator to check their device and internet connection: </span><ul><li dir="rtl"> Get the Administrator to troubleshoot using the steps above</li> </ul>  <span>Let us know if you still have issues with loading: </span><ul><li dir="rtl"> Report a problem by contacting us at admin@oppia.org. </li></ul>]]></string>
+  <string name="faq_answer_8"><![CDATA[ <span> To download an Exploration: </span> <br/><br/> <span>1. From the Home Page, tap on a Topic or Exploration. </span> <br/><br/> <span>2. From that Topic Page, tap on the <strong>Info</strong> tab. </span> <br/><br/> <span>3. Tap on <strong>Download Topic</strong>.  </span> <br/><br/> <span>4. Depending on your app settings, you may need Administrator approval or stable Wifi connection to complete your download. If needed, once these requirements are satisfied, the Topic has been downloaded onto the device and can be used offline by all profiles.  </span>]]></string>
```

Comment:
I'm not sure that I understand these changes--why are we switching from paragraphs to spans & breaks?

Reply:
`<p>` has extra spacing due to which there was double spacing after and before every new paragraph tag. I replaced this tag with `<span>` and used line breaks `<br/>` to give new line, as `<span>` tag was not adding any spacing.

You can check the screenshot below that used paragraphs tag.

<img width="144" alt="Screenshot_1627302732" src="https://user-images.githubusercontent.com/16301028/126989500-90c3fa29-f3b9-461b-b1ed-00e4c1b053f5.png">

Screenshot with spans and break tags.
<img width="144" alt="Screenshot_1627979944" src="https://user-images.githubusercontent.com/16301028/127987817-67e944ad-6751-4bca-af0a-5ca7ea7825ac.png">

Reply:
Do we have an understanding of why ``<p>`` introduces extra spacing in RTL mode? I think we need to understand that to see if there's a way to make the current formatting work before changing it. ``span`` doesn't actually introduce any visual changes to the document (or at least, it's not meant to unless there is styling to define it) unlike a paragraph or div. I want to better understand how paragraph is meant to behave and why it changes in RTL vs. LTR since that will provide us guidance with how to properly format this.

Do you have a specific page or reference that led you to choosing span here?

Reply:
Yes here is the link https://stackoverflow.com/questions/6874402/remove-space-above-and-below-p-tag-html and 
 https://www.w3schools.com/tags/tag_p.asp

Reply:
That doesn't necessarily explain why this happens in Android, though. Do we know why paragraphs have padding/margins on Android?